spec/clusterable/commitment_spec.rb

```diff
@@ -19,16 +19,16 @@
   end
 
   it "validates local_shelving_type" do
-    expect(comm.valid?).to be true
+    expect(comm).to be_valid
```

Comment:
with all these changes, rspec will better describe failures - rspec can dynamically generate matchers for any predicate ending in `?` that returns a boolean

Reply:
:exploding_head: